### PR TITLE
fix: HUMSIG-37: Visual fixes for Project settings

### PR DESCRIPTION
**Pull request by makseq**
## Annotation tab updated
image

## Annotation tab old
image

------

## Model tab new
image

## Model tab old
image

------

## Predictions tab new
image

## Predictions tab old
image

### PR fulfills these requirements
- [ ] Tests for the changes have been added/updated
- [ ] Docs have been added/updated
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance

### Describe the reason for change
The previous interface had visual flaws. 

### What feature flags were used to cover this change?
None

### This change affects (describe how if yes)
- [ ] Performance
- [ ] Security
- [x] UX

### Does this PR introduce a breaking change?
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [x] No
- [ ] Not sure (briefly explain the situation below)

### CODE REVIEW

**Comment by makseq** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8540366352)
> Successfully merged: Already up to date.

File: web/apps/labelstudio/src/pages/Settings/AnnotationSettings/ModelVersionSelector.js [hunk 1]
```diff
@@ -72,13 +72,10 @@ export const ModelVersionSelector = ({
...
-    <>
-      <Label
-        description={
-          <>Select which predictions or which model you want to use:</>
-        }
-      />
-
+    <div>
+      <p>
+          Select which predictions or which model you want to use:
```
Comment (hlomzik) [inline web/apps/labelstudio/src/pages/Settings/AnnotationSettings/ModelVersionSelector.js]: wrong indent
```suggestion
        Select which predictions or which model you want to use:
```

File: web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningList.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
-import { formatDistanceToNow, parseISO } from 'date-fns';
+import { formatDistanceToNow, format, parseISO } from 'date-fns';
 import { useCallback, useContext } from 'react';
 import { LsEllipsis } from '../../../assets/icons';
 
 import truncate from 'truncate-middle';
 import { Button, Dropdown, Menu } from '../../../components';
 import { confirm } from '../../../components/Modal/Modal';
 import { Oneof } from '../../../components/Oneof/Oneof';
+import {Tooltip} from "../../../components/Tooltip/Tooltip";
```
Comment (hlomzik) [inline web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningList.js]:
```suggestion
import { Tooltip } from "../../../components/Tooltip/Tooltip";
```

File: web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.js [hunk 3]
```diff
@@ -129,20 +129,19 @@ export const MachineLearningSettings = () => {
         <Divider height={32} />
 
         {backends.length > 0 && (
-          <Description style={{ marginTop: 0, maxWidth: 680 }}>
+          <Description style={{ marginTop: 0, maxWidth: 680, lineHeight:'1.5em' }}>
```
Comment (hlomzik) [inline web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.js]:
```suggestion
          <Description style={{ marginTop: 0, maxWidth: 680, lineHeight: "1.5em" }}>
```

File: web/apps/labelstudio/src/pages/Settings/PredictionsSettings/PredictionsSettings.js [hunk 4]
```diff
@@ -43,26 +44,19 @@ export const PredictionsSettings = () => {
...
-          </Description>
-        )}
-
         {loaded && versions.length > 0 && (
           <Elem name={'title-block'}>
             <Elem name={'title'}>Predictions List</Elem>
-            <Caption>
-              Each card is associated with separate model version.
-            </Caption>
+            <Description  style={{ marginTop: 14 }}>
```
Comment (hlomzik) [inline web/apps/labelstudio/src/pages/Settings/PredictionsSettings/PredictionsSettings.js]: move this to styles file

File: web/apps/labelstudio/src/pages/Settings/PredictionsSettings/PredictionsSettings.js [hunk 5]
```diff
@@ -43,26 +44,19 @@ export const PredictionsSettings = () => {
...
-            >
-              see the documentation
-            </a>
-            .
-          </Description>
-        )}
-
         {loaded && versions.length > 0 && (
           <Elem name={'title-block'}>
             <Elem name={'title'}>Predictions List</Elem>
```
Comment (hlomzik) [inline web/apps/labelstudio/src/pages/Settings/PredictionsSettings/PredictionsSettings.js]:
```suggestion
          <Elem name="title-block">
            <Elem name="title">Predictions List</Elem>
```

File: web/apps/labelstudio/src/pages/Settings/PredictionsSettings/PredictionsSettings.js (hunk 4 above)
Comment (hlomzik) [reply web/apps/labelstudio/src/pages/Settings/PredictionsSettings/PredictionsSettings.js]: and why do you have description inside `title-block`?

**Review by hlomzik — CHANGES_REQUESTED** [review]
minor comments

File: web/apps/labelstudio/src/pages/Settings/PredictionsSettings/PredictionsSettings.js (hunk 4 above)
Comment (hlomzik) [reply web/apps/labelstudio/src/pages/Settings/PredictionsSettings/PredictionsSettings.js]: ah, that's external component, then this style is good.
but title-block > description is still a little confusing

**Comment by makseq** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8554694192)
> Successfully merged:  32 files changed, 298 insertions(+), 1030 deletions(-)

**Comment by makseq** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8558118488)
> Successfully merged:  1 file changed, 4 insertions(+), 2 deletions(-)

**Review by hlomzik — APPROVED** [review]

**Pull request merged**